src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1466,5 +1497,108 @@ private void OnPixelAreaChanged(float x, float y, float z, float w)
         {
             PixelArea = new RelativeVector4(x, y, z, w);
         }
+
+        private class ImageLayout : LayoutItem
+        {
+            private bool adjustViewSize = false;
```

Comment:
it is not used.

Reply:
Thank you for your comment.
I have removed it.